docs/leap_sapi/admin.rst

```diff
@@ -1473,14 +1473,17 @@ Viewing the Status of Problem Submissions
 To view the status of problems submitted to a project's solvers, go to the
 **Problem Status** tab.
 
+.. note:: Up to 1000 of your newest problem submissions are stored and
+    accessible for up to 365 days.
```

Comment:
suggest "most recent" rather than "newest"

Reply:
This addition worries me for two reasons:

1. In the future, if storage parameters changes again, there's no reason to know that we are specifying problem-storage in this section. 
2. For most users of this page most times, this information is not useful and so lengthens the text without strong benefit